ProcessMaker/Providers/ProcessMakerServiceProvider.php

```diff
@@ -43,6 +43,12 @@ public function boot()
      */
     public function register()
     {
+
+        // Dusk, if env is appropriate
+        if ($this->app->environment('local', 'testing', 'development')) {
```

Comment:
Incorrect usage.  You need to pass in the diff environments with an array.  Please see: https://laravel.com/docs/5.6/configuration#determining-the-current-environment

Also, might be good to simply do:

if(!$this->app->environment('production')) {
...
}